app/lib/admin/tools/remove_publisher_and_block_all_members.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:pub_dev/account/backend.dart';
+import 'package:pub_dev/publisher/backend.dart';
+import 'package:pub_dev/publisher/models.dart';
+import 'package:pub_dev/shared/datastore.dart';
+
+Future<String> executeRemovePublisherAndBlockAllMembers(
+    List<String> args) async {
+  if (args.isEmpty ||
+      args.length > 2 ||
+      (args.length == 2 && args[1] != 'delete')) {
+    return 'Remove publisher and blocks all members.\n'
+        '  <tools-command> <publisherId> # list publisher data\n'
+        '  <tools-command> <publisherId> delete # remove publisher and block members\n';
```

Comment:
Might it not be safer to do:
```suggestion
        '  <tools-command> list <publisherId> # list publisher data\n'
        '  <tools-command> delete <publisherId> # remove publisher and block members\n';
```